lib/components/molecules/QuoteMedia/QuoteMedia.mock.js

```diff
@@ -1,11 +1,17 @@
-import Blockquote from '../../atoms/Blockquote/Blockquote';
+import Blockquote from '../Blockquote';
+import defaultBlockquote from '../Blockquote/Blockquote.mock';
 
 const defaultQuoteMedia = {
-  className: 'quoteMedia',
   HeadingType: 'h2',
-  title: 'Lorem ipsum dollor',
-  variant: 'stacked',
-  quoteContent: () => <Blockquote />,
+  title: 'Inspirational Quotes',
+  quoteContent: () => <Blockquote {...defaultBlockquote} />,
 };
 
-export default defaultQuoteMedia;
+const stackedQuoteMedia = {
+  HeadingType: 'h2',
+  title: 'Inspirational Quotes',
+  variation: 'stacked',
+  quoteContent: () => <Blockquote {...defaultBlockquote} />,
+};
+
+export { defaultQuoteMedia, stackedQuoteMedia };
```

Comment:
We should define two objects: defaultQuoteMedia  and stackedQuoteMedia and use their props on the basis of variations in story.